src/dialects/abstract/query-interface.js

```diff
@@ -242,8 +242,6 @@ export class QueryInterface {
       table: tableName,
       context: 'createTable',
       withoutForeignKeyConstraints: options.withoutForeignKeyConstraints,
-      // schema override for multi-tenancy
-      schema: options.schema,
```

Comment:
This has broken a postgres test, but I think we should update the test rather than leaving this in here

Reply:
I think the test is fine but this implementation needs to be changed slightly

Reply:
Surprisingly it's not failing when I run it locally 🤔

Reply:
wait why did I remove that it's super important until we've actually rewritten this API 🤦‍♀️